src/Sylius/Behat/Context/Api/Admin/ManagingExchangeRatesContext.php

```diff
@@ -0,0 +1,342 @@
+     * @Given /^I am editing (this exchange rate)$/
+     */
+    public function iWantToEditThisExchangeRate(ExchangeRateInterface $exchangeRate): void
+    {
+        $this->client->buildUpdateRequest('exchange_rates', $exchangeRate->getId());
+
+        $this->sharedStorage->set('exchange_rate_id', $exchangeRate->getId());
+    }
+
+    /**
+     * @Given I am browsing exchange rates of the store
+     * @When I browse exchange rates
+     * @When I browse exchange rates of the store
+     */
+    public function iBrowseExchangeRatesOfTheStore(): void
+    {
+        $this->client->index('exchange_rates');
+    }
+
+    /**
+     * @When I specify its ratio as :ratio
+     * @When I don't specify its ratio
+     */
+    public function iSpecifyItsRatioAs(?string $ratio = null): void
+    {
+        if ($ratio !== null) {
+            $this->client->addRequestData('ratio', $ratio);
+        }
+    }
+
+    /**
+     * @When I choose :currencyCode as the source currency
+     */
+    public function iChooseAsTheSourceCurrency(string $currencyCode): void
+    {
+        $this->client->addRequestData('sourceCurrency', '/new-api/currencies/' . $currencyCode);
+    }
+
+    /**
+     * @When I choose :currencyCode as the target currency
+     */
+    public function iChooseAsTheTargetCurrency(string $currencyCode): void
+    {
+        $this->client->addRequestData('targetCurrency', '/new-api/currencies/' . $currencyCode);
+    }
+
+    /**
+     * @When I (try to) add it
+     */
+    public function iAddIt(): void
+    {
+        $this->client->create();
+    }
+
+    /**
+     * @When I change ratio to :ratio
+     */
+    public function iChangeRatioTo(string $ratio): void
+    {
+        $this->client->updateRequestData(['ratio' => $ratio]);
+    }
+
+    /**
+     * @When I save my changes
+     */
+    public function iSaveMyChanges(): void
+    {
+        $this->client->update();
+    }
+
+    /**
+     * @When I delete the exchange rate between :sourceCurrency and :targetCurrency
+     */
+    public function iDeleteTheExchangeRateBetweenAnd(CurrencyInterface $sourceCurrency, CurrencyInterface $targetCurrency) : void
+    {
+        /** @var ExchangeRateInterface */
+        $exchangeRate = $this->getExchangeRateBetweenCurrencies($sourceCurrency->getCode(), $targetCurrency->getCode());
+
+        $this->client->delete('exchange_rates', $exchangeRate->getId());
+    }
+
+    /**
+     * @Then I should see :count exchange rates on the list
+     */
+    public function iShouldSeeExchangeRatesOnTheList(int $count): void
+    {
+        Assert::count($this->client->getCollectionItems(), $count);
+    }
+
+    /**
+     * @Then I should see a single exchange rate in the list
+     * @Then I should( still) see one exchange rate on the list
+     */
+    public function iShouldSeeASingleExchangeRateInTheList(): void
+    {
+        $this->client->index('exchange_rates');
+        Assert::count($this->client->getCollectionItems(), 1);
```

Comment:
Why not `countCollectionItems`?

Reply:
I changed it after : https://github.com/Sylius/Sylius/pull/11178#discussion_r389719701